### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by spuddybike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,27 +88,24 @@ usage: Options are as follows...
     sumstats=TRUE
     obsLimit=1000
 
+ -- format [2.5 | 3.2]
```
Comment (ollylucl) [inline README.md:91]: Is there an extra space before format? Should this be --format, not -- format? Also maybe have an extra blank line after this line, to be consistent with the existing formatting of the file. e.g.

--format [2.5 | 3.2]
<this is a new blank line>
--exclude <arg> (optional) exclude statistics for variables specified in the file with specified path. Format for the exlude file:

**Comment by spuddybike** [comment]
@ollylucl Could you merge the PR please

**Review by ollylucl — APPROVED** [review]
Looks good to me

**Review by ollylucl — APPROVED** [review]

**Comment by ollylucl** [comment]
@spuddybike I've given an approving review, but I'm not able to merge this PR, it still says that merging is blocked?

**Comment by spuddybike** [comment]
@ollylucl sorry - for some reason didn't give you sufficient rights - should be working now

**Pull request merged**